### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #153

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,29 @@ NCEPLIBS inquiries, contact [Edward
...
 [Alex Richert](mailto:alexander.richert@noaa.gov)).
 
-## Related NCEPLIBS Projects
+## Prerequisite External Projects
+
+Project | Notes
+-----------|------
+[libjasper.a](http://www.ece.uvic.ca/~mdadams/jasper/) | [JPEG-2000](http://www.jpeg.org/JPEG2000.html) library.
+[libpng](http://www.libpng.org/pub/png/libpng.html) | PNG compression library
+[libz](http://www.gzip.org/zlib/) | zlib compression library
```
Comment (AlexanderRichert-NOAA) [inline README.md]: URL should be https://www.zlib.net/ (and I would go with the official name zlib)

File: README.md [hunk 2]
```diff
@@ -15,12 +15,29 @@ NCEPLIBS inquiries, contact [Edward
 Hartnett](mailto:edward.hartnett@noaa.gov) (secondary point of contact
 [Alex Richert](mailto:alexander.richert@noaa.gov)).
 
-## Related NCEPLIBS Projects
+## Prerequisite External Projects
+
+Project | Notes
+-----------|------
+[libjasper.a](http://www.ece.uvic.ca/~mdadams/jasper/) | [JPEG-2000](http://www.jpeg.org/JPEG2000.html) library.
```
Comment (AlexanderRichert-NOAA) [inline README.md]: Unless it specifically has to be static jasper, then I'd change libjasper.a -> Jasper

File: README.md (hunk 1 above)
Comment (edwardhartnett) [reply README.md]: Thanks!

File: README.md (hunk 2 above)
Comment (edwardhartnett) [reply README.md]: Good point!

**Review by AlexanderRichert-NOAA — APPROVED** [review]

**Pull request merged**